### PR TITLE
Model post likes

### PR DESCRIPTION
**Pull request by sailysaily**
Hey Adrian the post_likes model is ready. Please check it and if it looks good merge it!

### CODE REVIEW

File: app/models/post_like.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class PostLike < ApplicationRecord
+  belongs_to :user, dependent: :destroy
```
Comment (agutierrezgit) [inline app/models/post_like.rb]: I am not sure about this step. If we do "user, dependent: :destroy" we are saying that when you destroy a PostLike also destroy the user. See point 4.1.2.4 :dependent from https://guides.rubyonrails.org/association_basics.html

File: app/models/post_like.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+  belongs_to :post, dependent: :destroy
```
Comment (agutierrezgit) [inline app/models/post_like.rb]: Same as above

Comment (sailysaily) [reply app/models/post_like.rb]: Tienes toda la razon! El depedent: :destroy tiene que ir en el user model No el post_like

**Comment by sailysaily** [comment]
Hey Adrian! I did this changes you can review them and if everything looks good please go ahead end merge!

**Pull request merged**